aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt

```diff
@@ -36,7 +36,7 @@ import software.amazon.smithy.rust.codegen.smithy.generators.client.FluentClient
 import software.amazon.smithy.rust.codegen.smithy.generators.client.FluentClientSection
```

Comment:
Just a general question: I took a look the SDK diff and it seems there are no breaking changes there with this code change. Did you verify this?

Reply:
There shouldn't be, it's just a class rename. I've now called out the rename in the commit message and PR description.